### ISSUE TITLE
The document of tf.nn.dynamic_rnn needs to be re-formatted

### ISSUE THREAD

**Issue by leckie-chn** [tensorflow/tensorflow#13560]
Hi,

Someone take a look at the document page of [tf.nn.dynamic_rnn](https://www.tensorflow.org/api_docs/python/tf/nn/dynamic_rnn). I think there are some formatting bugs in the second half of the web page.

**Comment by asimshankar** [comment]
@deannalinchen @wolffg : Seems like the empty newlines in https://github.com/tensorflow/tensorflow/blob/r1.3/tensorflow/python/ops/rnn.py#L483 are confusing the website generator?

Would either of you have some cycles to investigate?

**Comment by rafaelleru** [comment]
Hy, I was trying to solve this issue, but when I forked the project and download it my source code file is different of the file in the web, it doesn't have the empty newlines that @asimshankar comments. Can anyone tell me why this is happening?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by leckie-chn** [comment]
@tensorflowbutler this bug has been fixed.
